prov/usnic/src/usdf_cq.c

```diff
@@ -954,9 +716,9 @@ static struct fi_ops_cq usdf_cq_context_ops = {
 static struct fi_ops_cq usdf_cq_context_soft_ops = {
 	.size = sizeof(struct fi_ops_cq),
 	.read = usdf_cq_read_context_soft,
-	.readfrom = usdf_cq_readfrom_context_soft,
+	.readfrom = usdf_cq_readfrom_context,
```

Comment:
Actually, maybe this should just be disabled. It doesn't look like it does the right thing.